cmd/telemetrygen/pkg/metrics/worker.go

```diff
@@ -24,6 +24,7 @@ type worker struct {
 	aggregationTemporality AggregationTemporality       // Temporality type to use
 	exemplars              []metricdata.Exemplar[int64] // exemplars to attach to the metric
 	numMetrics             int                          // how many metrics the worker has to generate (only when duration==0)
+	enforceUnique          bool                         // if true, the worker will generate unique timeseries
 	totalDuration          time.Duration                // how long to run the test for (overrides `numMetrics`)
 	limitPerSecond         rate.Limit                   // how many metrics per second to generate
```

Comment:
Couldn't we just have a limit of 1 here?

Reply:
The limit wouldn't allow for load-test levels of telemetry to be generated for backend systems without an astronomical number of binaries running. This creates new timeseries to ensure we're sending unique values within the provided time limit.